enhancements/microshift/generic-device-plugin-for-microshift.md

```diff
@@ -0,0 +1,356 @@
+---
+title: generic-device-plugin-for-microshift
+authors:
+  - pmtk
+reviewers:
+  - "@DanielFroehlich, MicroShift PM"
+  - "@ggiguash, MicroShift contributor"
+  - "@pacevedom, MicroShift contributor"
+  - "@eslutsky, MicroShift contributor"
+  - "@copejon, MicroShift contributor"
+approvers:
+  - "@jerpeter1, MicroShift Staff Engineer"
+api-approvers:
+  - none
+creation-date: 2025-05-13
+last-updated: 2025-05-13
+tracking-link:
+  - https://issues.redhat.com/browse/OCPSTRAT-2031
+---
+
+# Generic Device Plugin for MicroShift
+
+## Summary
+
+[Generic Device Plugin](https://github.com/squat/generic-device-plugin) is a
+straightforward device plugin for devices not needing special initialization
```

Comment:
"Device plugin is a device plugin..." - can we explain this without recursion :).